baremetal/metal3machine_manager.go

```diff
@@ -207,10 +207,9 @@ func (m *MachineManager) SetPauseAnnotation(ctx context.Context) error {
 		if _, ok := annotations[bmh.PausedAnnotation]; ok {
 			m.Log.Info("BaremetalHost is already paused")
 			return nil
-		} else {
-			m.Log.Info("Adding PausedAnnotation in BareMetalHost")
-			host.Annotations[bmh.PausedAnnotation] = pausedAnnotationKey
 		}
+		m.Log.Info("Adding PausedAnnotation in BareMetalHost")
+		host.Annotations[bmh.PausedAnnotation] = pausedAnnotationKey
```

Comment:
why changing this ? Kashif did it this way so that we would not override the user's pause annotation

Reply:
lint complaint fix, the 'else' has no effect since the 'if' has a 'return'

Reply:
ah ok then fine by me